src/index.html

```diff
@@ -8,5 +8,39 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    
+    <div class="calendar calendar__start-day--fri calendar__month-length--29">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--fri calendar--month-length--29">
```
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/dea29cd4-5024-4f7f-9379-2efe14363c7c)